### PR TITLE
Fix some phpdoc errors reported by apidoc extension

### PR DESCRIPTION
**Pull request by rob006**
Before:

> 129 errors have been logged to [...]/yii2api/errors.txt

After:

> 81 errors have been logged to [...]/yii2api/errors.txt

### CODE REVIEW

File: framework/base/Component.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * Component is the base class that implements the *property*, *event* and *behavior* features.
  *
  * Component provides the *event* and *behavior* features, in addition to the *property* feature which is implemented in
- * its parent class [[Object]].
+ * its parent class [[\yii\base\Object|Object]].
```
Comment (cebe) [inline framework/base/Component.php]: is `Object` not resolved in `yii\base` namespace? if not I'd consider that a bug in apidoc generator.

**Review by cebe — APPROVED** [review]

File: framework/base/Component.php (hunk 1 above)
Comment (rob006) [inline framework/base/Component.php]: `object` is reserved keyword in PHP.

**Review by dynasource — APPROVED** [review]
useful improvements. Couldn't find any flaws. Well done.

**Review by samdark — APPROVED** [review]

**Comment by cebe** [comment]
Thank you!

**Pull request merged**